src/net.h

```diff
@@ -413,6 +413,7 @@ class CNode
     CBlock thinBlock;
     std::vector<uint256> thinBlockHashes;
     std::vector<uint64_t> xThinBlockHashes;
+    std::map<uint64_t, CTransaction> mapMissingTx;
```

Comment:
I think we're putting too much stuff in the nodes; this should be tracked in the RM based on requests.  But that's for later.